### PR TITLE
fix(react-badge): Remove white border around presence badge when on a dark background

### PR DESCRIPTION
**Pull request by behowell**
## Previous Behavior

The background color of presence badge bleeds around the edge of the icon due to antialiasing on the svg and element background.

image
image

## New Behavior

Inset the background using `padding: 1px; background-clip: content-box`. Add `margin: -1px` to the icon to offset the padding. 

This works because all presence icons have a border around the edge that is at least 1px wide (except size="tiny", where this fix is not applied).

image
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                       |    Baseline (minified/GZIP) |                          PR |                                                                                                                                                                                                  Change |
| :------------------------------------------------------------------------------------------------------ | --------------------------: | --------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr> | `31.904 kB`<br />`8.296 kB` | `32.109 kB`<br />`8.382 kB` | `205 B` <br />`86 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                     |        Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------- | --------------------------: |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>               | `23.512 kB`<br />`7.197 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr> | `24.416 kB`<br />`7.506 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3aecae7fc8af9dc0c4de35f58a61b76ab9e9a659'>3aecae7fc8af9dc0c4de35f58a61b76ab9e9a659</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">107</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">109</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/InfoButton-mount_base.html">15</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/InfoButton-mount.html">21</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/Avatar-mount_base.html">724</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/Avatar-mount.html">702</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/Button-mount_base.html">378</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/Button-mount.html">369</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/Field-mount_base.html">1296</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/Field-mount.html">1250</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProvider-mount_base.html">881</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProvider-mount.html">922</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProviderWithTheme-mount_base.html">108</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProviderWithTheme-mount.html">114</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">93</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">96</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">107</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">109</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/InfoButton-mount_base.html">15</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/InfoButton-mount.html">21</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/MakeStyles-mount_base.html">1130</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/MakeStyles-mount.html">1114</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/Persona-mount_base.html">2010</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/Persona-mount.html">1970</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/SpinButton-mount_base.html">1571</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27780/perf-test-react-components/SpinButton-mount.html">1539</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: apps/vr-tests-react-components/src/stories/PresenceBadge.stories.tsx [hunk 1]
```diff
@@ -1,16 +1,10 @@
 import { storiesOf } from '@storybook/react';
```
Comment (fabricteam) [inline apps/vr-tests-react-components/src/stories/PresenceBadge.stories.tsx:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 21 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27780&commitId=3d06fe314f391dac90bcaeefa8eba362676c3fd9&env=prod)

✅ There was 3 screenshots added, 0 screenshots removed, 1915 screenshots unchanged, 0 screenshots with different dimensions and 18 screenshots with visible difference.


  <details><summary><b>unknown</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|63| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|63| Changed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|3| Changed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|3| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|21| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|15| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|12| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|8| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|4| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|3| Changed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|3| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|3| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|21| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|15| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|12| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|8| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|4| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|3| Changed|
|PresenceBadge Converged - inverted background.default - Dark Mode.chromium.png|0| Added|
|PresenceBadge Converged - inverted background.default - High Contrast.chromium.png|0| Added|
|PresenceBadge Converged - inverted background.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>

**Review by khmakoto — APPROVED** [review]

File: apps/vr-tests-react-components/src/stories/PresenceBadge.stories.tsx [hunk 2]
```diff
@@ -40,12 +36,43 @@ storiesOf('PresenceBadge Converged - sizes', module).addStory(
...
+  .addStory(
+    'default',
+    () => (
+      <div
+        style={{
+          display: 'inline-grid',
+          gridTemplateColumns: `repeat(${2 * statuses.length}, auto)`,
+          placeItems: 'start',
+          gap: '10px',
+          padding: '16px',
```
[the remark this answers is not in the document]
Comment (behowell) [reply apps/vr-tests-react-components/src/stories/PresenceBadge.stories.tsx:61]: I don't think it's necessary to use spacing tokens for VR tests.

**Pull request merged**